data/configurables/tech_mappings.txt

```diff
@@ -0,0 +1,264 @@
+# Notes:
+# - Each Vic3 tech and invention is linked to zero or more HoI4 tech levels.
+# - More than one Vic3 tech or invention may grant a given HoI4 tech level.
+# - If a higher level is earned, it will be granted even if the lower level was not earned.
+# - Limits can use C++-style string literals (https://en.cppreference.com/w/cpp/language/string_literal) to include quotes
+
+
+# ========================
+# === HOI4 BASIC TECHS ===
+# ========================
+
+# Trivial mappings ensuring all nations receive these techs
+
+link = {
+	hoi4 = infantry_weapons
+	hoi4 = gw_artillery
+	hoi4 = tech_support
+}
+
+link = {
+	limit = R"(not = { has_dlc = "Man the Guns" })"
```

Comment:
I get it, but I don't like it. When faced with something like that, I predefine my own "keys", like 
```mantheguns = no```. There's only the two, anyway.

Reply:
Until the next expansion that splits things further. I also find I dislike putting strings into code unnecessarily, which something like that would do.